c10/util/BFloat16.h

```diff
@@ -7,7 +7,7 @@
 #include <cmath>
 #include <cstring>
 
-#if defined(__CUDACC__) && !defined(USE_ROCM)
+#if !(defined(USE_ROCM) || ((defined(CUDA_VERSION) && CUDA_VERSION < 11000) || (defined(__CUDA_ARCH__) && (__CUDA_ARCH__ < 800))))
```

Comment:
This change seems like it could be causing missing-include failures. Could you check if using the current include guard would work isntead?

Reply:
Made change